coalib/parsing/Globbing.py

```diff
@@ -169,7 +169,7 @@ def translate(pattern):
                 regex += '.*'
             # On Windows, '*' matches everything but the filesystem
             # separators '/' and '\'.
-            elif platform.system() == 'Windows':  # pragma posix: no cover
+            elif platform.system() == 'Windows':  # pragma: no cover
```

Comment:
This does not look like the right way to solve the problem. By changing `# pragma posix: no cover` to `# pragma: no cover` you are hiding the existing problem. Instead, we should find why code coverage was less than 100%.

Looks like it was because Appveyor build was failing, leading to this line not being tested.